controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -1433,6 +1446,21 @@ func (r *BareMetalHostReconciler) doServiceIfNeeded(prov provisioner.Provisioner
 		dirty = true
 	}
 
+	if hfcDirty && started {
+		hfcStillDirty, err := r.saveHostFirmwareComponents(prov, info, hfc)
+		if err != nil {
+			return actionError{errors.Wrap(err, "could not save the host firmware components")}
+		}
+
+		if hfcStillDirty {
+			info.log.Info("going to update the host firmware components")
```

Comment:
This is not super helpful, let's log the new components or drop the message.

Reply:
removing since previously we would call the `saveHostFirmwareComponents` and in case of changes to be save we will log the components.